src/components/TopBar/TopBar.vue

```diff
@@ -248,6 +280,10 @@ export default {
 				return !peer.sessionIds.length
 			} else return false
 		},
+
+		participantsInCall() {
+			return this.$store.getters.participantsInCall(this.token) ? this.$store.getters.participantsInCall(this.token) : ''
```

Comment:
```suggestion
			return this.$store.getters.participantsInCall(this.token) || ''
```
Shorter and not calling the getter twice?

Reply:
that returns a boolean

Reply:
### Firefox
![Bildschirmfoto vom 2022-11-22 11-16-19](https://user-images.githubusercontent.com/213943/203288414-5277e80e-3381-4a68-8a3b-3a0bb1c57240.png)

### Chromium
![Bildschirmfoto vom 2022-11-22 11-16-34](https://user-images.githubusercontent.com/213943/203288416-d314549b-38dc-4a50-9df6-7408b7b4b212.png)